bazel/external/bfsde.BUILD

```diff
@@ -84,3 +85,46 @@ pkg_tar(
     package_dir = "/usr",
     strip_prefix = "barefoot-bin",
 )
+
+# This string setting is templated with the correct version string by reading
+# the $SDE_INSTALL/share/VERSION file. Then one of the config settings below
+# will match and can be used with select().
+string_setting(
+    name = "sde_version_setting",
+    build_setting_default = "{SDE_VERSION}",
+)
```

Comment:
I actually don't know how to set a non default value. `string_flags` are exposed to the CLI, this one is **NOT**.